blog/archive.md

```diff
@@ -29,10 +29,7 @@ title: Blog archives
         {% endif %} -->
       </div>
     {% endfor %}
-    <!--<hr />
-    <p class="overline">Want to stay up to date with the latest news from Strati? <a
-      href="#">Subscribe</a> to the blog!
-    </p>-->
+    <!--<hr /> -->
```

Comment:
What is this doing? Can this line be entirely deleted?

Reply:
Yeah, I think so.